Fix CI

fpereira1 opened the pull request:
Fixed CI. Also bumped all dependencies for the example that was failing. Will attempt to go through and do the same for all others at some point

from: 17 vulnerabilities (8 low, 8 moderate, 1 high)
to: 5 moderate severity vulnerabilities

fpereira1 committed:
nvmrc update

fpereira1 committed:
npm audit fix

fpereira1 committed:
minor bumps

fpereira1 committed:
chore: update @testing-library/react to 16.3.0

fpereira1 committed:
Update Jest assertions to use toHaveBeenCalled instead of toBeCalled

fpereira1 committed:
minor bump react dependencies

fpereira1 committed:
bump typescript

fpereira1 committed:
other bumps

fpereira1 committed:
minor updates

fpereira1 committed:
trigger build --ci

fpereira1 committed:
trigger build --ci

fpereira1 committed:
github actions bumps

fpereira1 committed:
updates based on docs

fpereira1 committed:
update HelloWorld.test.tsx

christian-bromann commented:
@fpereira1 I've send you an invite to the org, so hopefully I don't need to approve the workflows anymore.

fpereira1 committed:
trigger build --ci

fpereira1 committed:
updates

fpereira1 committed:
trigger build --ci

fpereira1 committed:
npm audit fix

fpereira1 committed:
Fix wdio tests and improve test isolation

- Reduced logging verbosity from 'info' to 'error' in wdio.conf.ts
- Fixed test isolation by scoping DOM queries to component container
- Added cleanup() after each test to prevent state leaking between tests
- All existing tests now pass with improved isolation

fpereira1 commented:
@christian-bromann this isn't a fully passing build (there was a lot broken in this repo) but I think there are some good changes here and I'd like to see this work reviewed/merged.  Thank you

christian-bromann commented:
What do you think is still broken?

fpereira1 commented:
I was hoping to get CI passing in this PR but it's going to take more work
to get everything passing, in this PR, I fixed some things but not all 🙂

On Wed, Dec 3, 2025, 7:40 AM Christian Bromann ***@***.***>
wrote:

> *christian-bromann* left a comment
> (webdriverio/component-testing-examples#31)
> <https://github.com/webdriverio/component-testing-examples/pull/31#issuecomment-3603468303>
>
> What do you think is still broken?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/webdriverio/component-testing-examples/pull/31#issuecomment-3603468303>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGUAR4P7JT7NVHNIQBSEH337XMKLAVCNFSM6AAAAACHT6PZKSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMBTGQ3DQMZQGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
